src/components/Environment/EnvironmentListView.tsx

```diff
@@ -123,53 +86,26 @@ const EnvironmentListView: React.FC<Props> = ({
       ) : (
         <>
           {description}
-          <Toolbar
-            collapseListedFiltersBreakpoint="xl"
-            clearAllFilters={onClearAllFilters}
-            clearFiltersButtonText="Clear filters"
-          >
+          <Toolbar collapseListedFiltersBreakpoint="xl">
             <ToolbarContent className="pf-u-pl-0">
-              {environments.length > 0 ? (
```

Comment:
@abhinandan13jan Do you know if the environment filter will be back again after MVP? 


Asking this because if its going to come back post mvp then  we could hide this filter using on the mvp feature flag rather than removing the code itself. @rohitkrai03 any thoughts on this?

Reply:
yes filtering will be back when the user can create multiple environments. This is just to be removed for mvp

Reply:
added as MVP feature flag